api4/team_test.go

```diff
@@ -3352,19 +3352,22 @@ func TestInvalidateAllEmailInvites(t *testing.T) {
 	defer th.TearDown()
 
 	t.Run("Forbidden when request performed by system user", func(t *testing.T) {
-		_, res := th.Client.InvalidateAllEmailInvites()
+		ok, res := th.Client.InvalidateEmailInvites()
+		require.Equal(t, ok, false)
```

Comment:
Very minor detail but I've been hit by this in the past while debugging test failures:
these assertions assume the first value passed is the *expected value* while the second is the *actual value* we want to verify. This makes a difference in the final test output in case of a failure so I'd probably swap them.